package.json

```diff
@@ -33,51 +34,20 @@
     "type": "git",
     "url": "https://github.com/devtools-html/perf.html"
   },
-  "devDependencies": {
+  "dependencies": {
     "array-move": "^1.0.0",
     "array-range": "^1.0.1",
-    "babel-cli": "^6.24.0",
-    "babel-core": "^6.24.0",
-    "babel-eslint": "^7.2.1",
-    "babel-loader": "^6.4.1",
     "babel-plugin-transform-runtime": "^6.23.0",
```

Comment:
The doc for this says: install `babel-plugin-transform-runtime` as dev dependency, and `babel-runtime` as prod dependency.